tools/serverpod_cli/lib/src/generator/generator.dart

```diff
@@ -9,6 +9,7 @@ import 'package:serverpod_cli/src/util/protocol_helper.dart';
 Future<bool> performGenerate({
   bool dartFormat = true,
   String? changedFile,
+  bool genOpenApi = false,
```

Comment:
I'm thinking here we may want to add an enum with generation types. I.e. currently `Dart` and `OpenAPI`, then pass in a `Set` of languages we want to generate for.